src/Discord.Net.Core/Entities/Users/IUser.cs

```diff
@@ -34,6 +46,16 @@ public interface IUser : ISnowflakeEntity, IMentionable, IPresence
         /// </returns>
         string GetAvatarUrl(ImageFormat format = ImageFormat.Auto, ushort size = 128);
         /// <summary>
+        ///     Gets the banner URL for this user.
+        /// </summary>
+        /// <param name="format">The format to return.</param>
+        /// <param name="size">The size of the image to return in. This can be any power of two between 16 and 2048.
+        /// </param>
+        /// <returns>
+        ///     A string representing the user's avatar URL; <c>null</c> if the user does not have an banner in place.
+        /// </returns>
+        string GetBannerUrl(ImageFormat format = ImageFormat.Auto, ushort size = 128);
```

Comment:
since banners are bigger images, shouldn't the default size be bigger?

also is the size of the image the width or height?